wonderland/src/index.js

```diff
@@ -20,7 +15,7 @@ WL.registerComponent('zesty-ad', {
     /* Your ad space index */
     adSpace: {type: WL.Type.Int},
     /* The default ad format, determines aspect ratio */
-    defaultAd: {type: WL.Type.Enum, values: ['tall', 'wide', 'square'], default: 'square'},
+    adFormat: {type: WL.Type.Enum, values: ['tall', 'wide', 'square'], default: 'square'},
```

Comment:
You could use `Object.keya(formats)` to have a single source of truth for the possibilities.